src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java

```diff
@@ -1,15 +1,12 @@
 package com.dev.cinema.model.dto;
 
-import com.dev.cinema.model.Ticket;
-import com.dev.cinema.model.User;
-import java.time.LocalDateTime;
 import java.util.List;
 
 public class OrderResponseDto {
     private Long id;
-    private List<Ticket> tickets;
-    private LocalDateTime orderDate;
-    private User user;
+    private List<Long> ticketsId;
+    private String orderDate;
```

Comment:
could be LocalDateTime

Reply:
for request String, for a response can be Object, fixed